src/rs.cpp

```diff
@@ -3951,3 +3951,27 @@ float rs2_calculate_target_z(rs2_device* device, rs2_frame_queue* queue1, rs2_fr
     }
 }
 HANDLE_EXCEPTIONS_AND_RETURN(-1.f, device, queue1, queue2, queue3, target_width, target_height)
+
+
+rs2_safety_preset* rs2_get_safety_preset_at_index(const rs2_device* device, int index, rs2_error** error) BEGIN_API_CALL
+{
+    VALIDATE_NOT_NULL(device);
```

Comment:
index validation is normally checked on this file or inside the called functions?
We should check 0-63 somewhere (and verify on UT)

Reply:
Oh just saw your next comment :)

Reply:
Add `VALIDATE_RANGE(index, 0, 63)`